nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -131,6 +131,15 @@
       and <option>services.limesurvey.virtualHost</option> options.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The setopt declarations will be evaluated at the end of zshrc, so any code in interactiveShellInit,
+      loginShellInit and promptInit may break if it relies on those options being set.
```

Comment:
Instead of `loginShellInit` you can write `<xref linkend="opt-programs.zsh.loginShellInit" />` to link the option's docs here. This shold be done for `interactiveShellInit` and `promptInit` as well.

Reply:
Thanks, updated PR.